code/modules/admin/admin_secrets.dm

```diff
@@ -28,7 +28,6 @@ var/datum/admin_secrets/admin_secrets = new()
 	var/list/datum/admin_secret_item/items
 
 /datum/admin_secret_category
-	..()
```

Comment:
You cant call a super on something that isnt a proc

Reply:
You cant tell me what to do.

Reply:
The renderer can